scapy/contrib/isotp.uts

```diff
@@ -1031,12 +1048,118 @@ def sender():
 
 sender()
 
++ ISOTPSoftSocket MITM attack tests
+
+= bridge and sniff with isotp soft sockets set up vcan0 and vcan1 for package forwarding vcan1
+~ linux needs_root
+
+def forwarding(pkt):
+    return pkt
+
+def bridge():
+    with ISOTPSoftSocket(new_can_socket0(), sid=0x641, did=0x241) as bSocket0, \
+         ISOTPSoftSocket(new_can_socket1(), sid=0x241, did=0x641) as bSocket1:
+        bridge_and_sniff(if1=bSocket0, if2=bSocket1, xfrm12=forwarding, xfrm21=forwarding, timeout=0.5)
+
+threadBridge = threading.Thread(target=bridge)
+threadBridge.start()
+
+with ISOTPSoftSocket(new_can_socket0(), sid=0x241, did=0x641) as isoTpSocket0, \
+     ISOTPSoftSocket(new_can_socket1(), sid=0x641, did=0x241) as isoTpSocket1:
+    time.sleep(0.2)
+    packet = ISOTP('Request')
+    isoTpSocket0.send(packet)
+    packetsVCan1 = isoTpSocket1.sniff(timeout=0.5)
```

Comment:
You should use the `started_callback` parameter of sniff, to send the packet.
`packetsVCan1 = isoTpSocket1.sniff(timeout=0.5, started_callback=lambda: isoTpSocket0.send(packet))`